webapp-django/crashstats/base/views.py

```diff
@@ -3,6 +3,16 @@
 from django.shortcuts import render
 
 
+class HttpResponseBadRequest(http.HttpResponseBadRequest):
```

Comment:
Yeah. I was thinking about that too. If that was your almost immediate reaction too, it's an indicator that you're right.